src/components/Authentication/OAuth.vue

```diff
@@ -0,0 +1,73 @@
+<template lang="pug">
+.oauth-widget(v-if="providers.length")
+  v-container
+    h4.ml-0 Or {{verb}} with OAuth
+    a.no-decorate(v-for="provider in providers",
```

Comment:
`v-btn` has an `href` property, so we can get rid of this element and the `no-decorate` class.

```javascript
    v-btn(v-for="provider in providers",
        :key="provider.id",
        :href="provider.url",
        :dark="iconMap[provider.id].dark", :color="provider.id")
      v-icon.aligned {{ $vuetify.icons[iconMap[provider.id].icon] }}
      .ml-2 {{ provider.name }}
```